src/Application/EudonetParis/Command/ImportEudonetParisRegulationCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\EudonetParis\Command;
+
+use App\Application\CommandInterface;
+use App\Application\Regulation\Command\SaveRegulationGeneralInfoCommand;
+use App\Domain\EudonetParis\EudonetParisLocationItem;
+
+final class ImportEudonetParisRegulationCommand implements CommandInterface
+{
+    public function __construct(
+        public readonly SaveRegulationGeneralInfoCommand $generalInfoCommand,
+        /** @var EudonetParisLocationItem[] */
+        public readonly array $locationItems,
+    ) {
```

Comment:
```suggestion
    ) {
    $generalInfoCommand->source = RegulationOrderRecordSourceEnum::EUDONET_PARIS;
```

L'init de la source dans le constructeur de la commande me parait plus juste que dans le handler